### DISCUSSION TITLE
Getting Email logins are disabled, when is it enabled in Auth

### DISCUSSION THREAD

**Discussion by MarkusWiland** [supabase/supabase#7284] [Questions]
Hi, I have followed the steps on supabase page with next js, and taken the code and trying to login with email but when I do, I get back
code 400 Email logins are disabled.

Anyone what is the problem?

**Comment by jwright04** [comment]
I'm hoping by now you have resolved this but in case any one else comes across this issue.  If you receive the `Email logins are disabled` message, all you have to do is enable it in your Settings.  Go to... 

1. Supabase project dashboard.
2. On the left side click `Settings`
3. Scroll down to `Auth Providers`
4. You will see that `Email` is disabled, just click to enable it
5. Select your options
6. Click Save

**Comment by KnudH** [comment]
I get the same error but in selfhosted environment. How to solve it there?

**Reply by taujin** [reply]
Self-hosted here too.

You might of set the env variable for email signup to false. Make sure this is set to true.
```
## Email auth
ENABLE_EMAIL_SIGNUP=true
```
This variable will affect the user's ability to sign in with email.

**Reply by KnudH** [reply]
Thanks! I already figured out and though about contributing to the docs, so it's easier for future users. But my PR gets ignored for 3 months. See here for more info and better docs: https://github.com/supabase/supabase/pull/20773
